### DISCUSSION TITLE
Two accounts in app Auth

### DISCUSSION THREAD

**Discussion by ghost** [ente-io/ente#2510] [Enhancements]
Add the ability to use two Ente Auth accounts in the same app so you can switch between them quickly

**Comment by finnjosten** [comment]
Please add this as i have one for personal and my work and it would be so awesome if i could easily switch between them without having to logout and back in

**Comment by nathfavour** [comment]
yeah, very useful feature

**Comment by gxaris** [comment]
That would be the one thing that separates this app with the rest. I upvote this idea, i believe that would solve many user's issues with separating personal and work otp

**Comment by Banana-97** [comment]
This is the only thing stopping me from using Ente.. Hope they can roll this function out soon.

**Comment by x80486** [comment]
This is currently possible while using the newly introduced tags.

**Reply by Banana-97** [reply]
No, this is totally different. The main purpose is to have it separated into different accounts. Eg: 1 Personal, 1 Work, 1 Family
By having it all in the same account, if you would need to share the account, you would end up be sharing all of your OTP's even personals.

This is one of the great feature in Google Authenticator and the only one stopping people to switch to others.

**Comment by LeGenDaRy15PT** [comment]
It would be great if they implemented multiple accounts, I also come from Google Authenticator and I'm also missing that feature in this app.

**Comment by Bl4ckspell7** [comment]
Yes, please add the possibility to save multiple accounts and easily switch between them. I have a personal and a work account.

**Comment by kaizokan** [comment]
Maybe separate the Photos and Auth account first? Will prevent deleting both when deleting one, confusion, server costs for the unused Photos storage space or processing and hopefully remove the inactivity policy for Auth.

**Comment by Bl4ckspell7** [comment]
Any plans on implementing multi accounts support?

**Comment by nathfavour** [comment]
I'd say we give them more time.
Ente seems to be a startup with a huge ambition on privacy, and these
things take time, trust me.

In the meantime I will fork the flutter repo and try playing around with a
mockup implementation :)

**Comment by conorseed** [comment]
+1 for this. Have a personal and a work account, and it's painful having to switch between the two constantly. 

An alternative is to enable add/remove functionality to the web interface, so I can use the web interface for work.

**Comment by the-AO** [comment]
+1

**Comment by lcdss** [comment]
+1000 for the ability to switch between multiple accounts. I have my account and also have to access my father's account to help him when necessary.

**Comment by alexmccart** [comment]
I would also like this feature added. One account, two emails.

**Comment by zohaibadnan137** [comment]
+1 for multiple account support. This is the only reason why I haven't been able to switch to Ente for both personal and work usage.

**Comment by RajStartsHere** [comment]
+1

**Comment by jeanoderco** [comment]
+1, extremally important feature

**Comment by PS222407** [comment]
+1

**Comment by benf2004** [comment]
I made a pull request to add this feature here:

https://github.com/ente/ente/pull/11686

You can add up to 5 accounts (signed in or local-only).

**Comment by audifan88** [comment]
+1, really want to separate work and personal items

**Comment by NeeL21YT** [comment]
@ente please update on this
